ansible/roles/satellite_master/tasks/main.yml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2015 TWO SIGMA OPEN SOURCE, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+- name: copy satellite master jar
+  copy: src={{ satellite_repo_root }}/satellite-master/target/{{ master_jar }} dest={{ satellite_install_dir }}/
```

Comment:
Instead of relativizing these paths, since there's just the 3 files (jar, riemann config, satellite config), make those things have an explicit path, so that the full, clear paths are in the vars.yml. Then, in the vars.yml, you can still use a few variables to simplify/reduce redunancy, but anything that you'd want to change will be in one place.